src/LibraryManager.Vsix/Json/Completion/CompletionController.cs

```diff
@@ -151,10 +151,10 @@ private void OnCommitted(object sender, EventArgs e)
             }
         }
 
-        private JSONParseItem GetItemBeforePosition(int pos, JSONEditorDocument document)
+        private JSONParseItem GetItemBeforePosition(int pos, JSONComplexItem document)
```

Comment:
>document [](start = 77, length = 8)

This is no longer taking a document, so it should be renamed.

Reply:
Will rename